resources/views/Checkout/AmazonPayWidgetsReinit.twig

```diff
@@ -0,0 +1,64 @@
+<script type="text/javascript">
+    $(document).ready(function () {
+        amazon.Login.setClientId('{{ content.clientId }}');
+
+        var $orderReferenceId;
+        addressBookWidget = new OffAmazonPayments.Widgets.AddressBook({
+            sellerId: '{{ content.sellerId }}',
+            scope: '{{ content.addressBookScope }}',
+            onOrderReferenceCreate: function (orderReference) {
+
+                // Here is where you can grab the Order Reference ID.
+                $orderReferenceId = orderReference.getAmazonOrderReferenceId();
+            },
+            onAddressSelect: function (orderReference) {
+
+                $.post("/payment/payone/checkout/amazonPay/getOrderReferenceReinit/{{ orderId }}",
+                    {
+                        "accessToken": "{{ accessToken }}",
+                        "workOrderId": "{{ workOrderId }}",
+                        "amazonReferenceId": $orderReferenceId
+                    },
+                    function (response) {
+
+                    });
+
```

Comment:
empty lines

Reply:
I removed the empty function, there s no need for the response